templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete Driver?</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+
+    <input type="submit" value="Yes" class="btn btn-danger">
```

Comment:
Add a `Cancel` button apart from `Yes` as mentioned in the checklist. It should lead to a previous page.

Reply:
Done, thanks